src/components/Header.js

```diff
@@ -70,12 +147,12 @@ export function DriverCode({ languages, selectedLanguage, actualLanguage }) {
 
   return (
     <DriverCodeBox>
-      <img src={githubIcon} alt='GitHub' />
-      <DriverCodeText 
+      <DriverCodeLink
         href={languages[driver].url} 
         target='_blank'>
-        Driver Code
-      </DriverCodeText>
+        <DriverCodeIcon src={githubIcon} alt='GitHub' />
```

Comment:
alt tag should describe the link target in "linked images"
example: "_Sourcecode of the {languages[driver].name} driver_"